checker/tests/interning/Arrays.java

```diff
@@ -33,6 +33,7 @@ public static void test(@Interned Integer i, @Interned String s) {
         @Interned String @Interned [][] multi2 = new @Interned String @Interned [2][];
     }
 
+    // :: error: (super.invocation.invalid)
```

Comment:
Does this error make sense for the Interning Checker?
Won't this always be created for every `@Interned class`, because `java.lang.Object`'s constructor is not `@Interned`?
Would an alternative be to mark the j.l.Object constructor `@Interned`?

A discussion of `@Interned` on a class declaration and constructor would be very helpful in the manual to understand the intended semantics for this system.

Reply:
Yes, this would be a problem with every `@Interned class`:  human investigation would be needed.
I was thinking that was acceptable, because using an object as interned needs consideration from a person.

I hadn't thought of the alternative of marking the `java.lang.Object` constructor `@Interned`.  That would certainly eliminate the warnings, with the pros and cons attending to that.  I think I like that suggestion.